src/hw_stats.py

```diff
@@ -14,11 +14,47 @@
 
 # Copied from /include/constants.py file
 # Enlists components at core, tile, and node levels
-hw_comp_energy = {'xbar_mvm':param.xbar_ip_pow_dyn*param.xbar_ip_lat, 'xbar_op':param.xbar_op_pow_dyn*param.xbar_op_lat,
-                  'xbar_mtvm':param.xbar_ip_pow_dyn*param.xbar_ip_lat,
+hw_comp_energy = {'xbar_mvm':{  '100':param.xbar_ip_energy_dict['100'], \
+                                '90': param.xbar_ip_energy_dict['90'], \
+                                '80': param.xbar_ip_energy_dict['80'], \
+                                '70': param.xbar_ip_energy_dict['70'], \
+                                '60': param.xbar_ip_energy_dict['60'], \
+                                '50': param.xbar_ip_energy_dict['50'], \
+                                '40': param.xbar_ip_energy_dict['40'], \
+                                '30': param.xbar_ip_energy_dict['30'], \
+                                '20': param.xbar_ip_energy_dict['20'], \
+                                '10': param.xbar_ip_energy_dict['10']}, \
+                  'xbar_op':{  '100': param.xbar_ip_energy_dict['100'], \
+                                '90': param.xbar_ip_energy_dict['90'], \
+                                '80': param.xbar_ip_energy_dict['80'], \
+                                '70': param.xbar_ip_energy_dict['70'], \
+                                '60': param.xbar_ip_energy_dict['60'], \
+                                '50': param.xbar_ip_energy_dict['50'], \
+                                '40': param.xbar_ip_energy_dict['40'], \
+                                '30': param.xbar_ip_energy_dict['30'], \
+                                '20': param.xbar_ip_energy_dict['20'], \
+                                '10': param.xbar_ip_energy_dict['10']}, \
+                  'xbar_mtvm':{ '100':param.xbar_ip_energy_dict['100'], \
+                                '90': param.xbar_ip_energy_dict['90'], \
+                                '80': param.xbar_ip_energy_dict['80'], \
+                                '70': param.xbar_ip_energy_dict['70'], \
+                                '60': param.xbar_ip_energy_dict['60'], \
+                                '50': param.xbar_ip_energy_dict['50'], \
+                                '40': param.xbar_ip_energy_dict['40'], \
+                                '30': param.xbar_ip_energy_dict['30'], \
+                                '20': param.xbar_ip_energy_dict['20'], \
+                                '10': param.xbar_ip_energy_dict['10']}, \
```

Comment:
Can we remove the additions to dictionary for `xbar_op` and `xbar_mtvm` keys, since they aren't part of inference ?